planet.js

```diff
@@ -832,4 +753,4 @@ planet.connect = async (rn, {online = true} = {}) => {
   return s;
 }; */
 
-// planet.connect('lol');
+planet.connect('lol');
```

Comment:
If anything this should use planet.connect in edit.js.

Either way it shouldn't be enabled by default.